config/test.secret.example.exs

```diff
@@ -4,3 +4,11 @@ use Mix.Config
 # config :movie_wager_api, MovieWagerApi.Repo,
 #   username: "postgres",
 #   password: "postgres"
+
+# Replace with your twitter app credentials
```

Comment:
I was actually having test failures when I didn't have this... How would I stub that information?